sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -1406,6 +1413,47 @@ Attempting to sample a standard sampled image with `sample_mipmap` or any other
 defined sampling function will result in undefined behaviour.
 ====
 
+==== `gather_image`
+
+[NOTE]
+=====
+`gather_image` is only currently supported in the Cuda backend.
+=====
+
+`DataT` is a vector type with four components. DataT can only be `sycl::float4`
+, `sycl::int4`, or `sycl::uint4`. `ImageHandle` must be a two dimensional image
+instantiated using an `image_descriptor` with non-zero `width` and `height`
+members. The `image_descriptor::depth` member must be zero.
+The return values, `DataT`, consist of the four texture, `T`, coordinates that
+would be used for linear filtering of `ImageHandle` at the coordinate specified
+by `Coords`, and for channel component `Component`.
+
+These coordinates are returned as follows:
+
+- `DataT.x = T[i, j + 1]`
+- `DataT.y = T[i + 1, j + 1]`
+- `DataT.z = T[i + 1, j]`
+- `DataT.w = T[i, j]`
+
+where `i` and `j` are calculated depending on the
+`sycl::coordinate_normalization_mode` used to constuct `ImageHandle` via a
+`bindless_image_sampler`.
+For `sycl::coordinate_normalization_mode::unnormalized` (in units of the image
+discretization) they are calculated as:
+
+- `i = floor(Coords.x - 0.5)`
+- `j = floor(Coords.y - 0.5)`
+
+In the Cuda backend there are different restrictions on the sizes of an image
+that can be created for a given device when `gather_image` is being used
+(Consult the latest backend specific documentation for more information).
+As such, for guaranteed correctness in the Cuda backend, it is also necessary
```

Comment:
Please rewrite this slightly, so that the document only specifies what is required of the SYCL user, and don't put the CUDA remarks into the normative section of the extension. Instead, you can put the CUDA information into a note, because notes are non-normative.

Reply:
OK sure, will do, thanks.